diff --git a/Interface/AddOns/alaChat_Classic/addon.lua b/Interface/AddOns/alaChat_Classic/addon.lua
index 741b246..4340fc4 100644
--- a/Interface/AddOns/alaChat_Classic/addon.lua
+++ b/Interface/AddOns/alaChat_Classic/addon.lua
@@ -143,20 +143,20 @@ local default = {
 	_version				 = 190830.0,
 	_overrideVersion		 = 190925.0,
 
-	position				 = "BELOW_EDITBOX",
+	position				 = "ABOVE_EDITOBX",
 	direction				 = "HORIZONTAL",
 	scale					 = 1.0,
 	alpha					 = 1.0,
-	barStyle				 = "ala",
+	barStyle				 = "blz",
 
 	shortChannelName		 = true,
 	shortChannelNameFormat	 = "NW",
 	hyperLinkEnhanced		 = true,
 	chatEmote				 = true,
 	ColorNameByClass		 = true,
-	shamanColor				 = false,
+	shamanColor				 = true,
 	filterQuestAnn			 = false,
-	channelBarChannel		 = { true,true,true,true,true,true,true,true,true,true,true,true,true,true },
+	channelBarChannel		 = { true,true,true,false,false,true,true,false,false,false,false,false,false,false },
 	channelBarStyle			 = "CHAR",
 
 	channel_Ignore_Switch 	 = true,
@@ -173,14 +173,14 @@ local default = {
 	DBMCountDown			 = true,
 	ReadyCheck				 = true,
 	statReport				 = true,
-	level					 = false,
+	level					 = true,
 	copy					 = true,
 	copyTagColor			 = { 0.25, 0.25, 1.00 },
 	copyTagFormat			 = "#s";
 
 	--hideConfBtn				 = true,
 	editBoxTab				 = true,
-	restoreAfterWhisper		 = false,
+	restoreAfterWhisper		 = true,
 	restoreAfterChannel		 = false,
 	hyperLinkHoverShow		 = true,
 
