diff --git a/Interface/AddOns/Bagnon/common/Wildpants/core/settings.lua b/Interface/AddOns/Bagnon/common/Wildpants/core/settings.lua
index fba2c05..09b7567 100644
--- a/Interface/AddOns/Bagnon/common/Wildpants/core/settings.lua
+++ b/Interface/AddOns/Bagnon/common/Wildpants/core/settings.lua
@@ -101,7 +101,7 @@ function Addon:StartupSettings()
 
 		resetPlayer = true,
 		displayBank = true, closeBank = true, displayAuction = true, displayGuild = true, displayMail = true, displayTrade = true, displayCraft = true, displayScrapping = true,
-		flashFind = true, tipCount = true, fading = true,
+		flashFind = true, tipCount = false, fading = true,
 
 		glowAlpha = 0.5,
 		glowQuality = true, glowNew = true, glowQuest = true, glowSets = true, glowUnusable = true,
