diff --git a/Interface/AddOns/DBM-Core/DBM-Core.lua b/Interface/AddOns/DBM-Core/DBM-Core.lua
index 344c04e..eea7acd 100644
--- a/Interface/AddOns/DBM-Core/DBM-Core.lua
+++ b/Interface/AddOns/DBM-Core/DBM-Core.lua
@@ -126,10 +126,10 @@ DBM.DefaultOptions = {
 	SpecialWarningSound4 = "Interface\\AddOns\\DBM-Core\\sounds\\ClassicSupport\\HoodWolfTransformPlayer01.ogg",
 	SpecialWarningSound5 = "Interface\\AddOns\\DBM-Core\\sounds\\ClassicSupport\\LOA_NAXX_AGGRO02.ogg",
 	ModelSoundValue = "Short",
-	CountdownVoice = "Corsica",
-	CountdownVoice2 = "Kolt",
-	CountdownVoice3 = "Smooth",
-	ChosenVoicePack = "None",
+	CountdownVoice = "VP:Yike",
+	CountdownVoice2 = "VP:Yike",
+	CountdownVoice3 = "VP:Yike",
+	ChosenVoicePack = "Yike",
 	VoiceOverSpecW2 = "DefaultOnly",
 	AlwaysPlayVoice = false,
 	EventSoundVictory2 = "None",
