diff --git a/Interface/AddOns/GlobalIgnoreList/GIL_UI.lua b/Interface/AddOns/GlobalIgnoreList/GIL_UI.lua
index bbb01e1..8b27cee 100644
--- a/Interface/AddOns/GlobalIgnoreList/GIL_UI.lua
+++ b/Interface/AddOns/GlobalIgnoreList/GIL_UI.lua
@@ -237,7 +237,7 @@ local function CreateFilterButtons()
 		end
 
 		FilterScrollButtons[count]:SetSize(BUTTON_WIDTH, BUTTON_HEIGHT)
-		FilterScrollButtons[count]:RegisterForClicks("LeftButtonUp")
+		FilterScrollButtons[count]:RegisterForClicks("LeftButtonUp", "RightButtonUp")
 		FilterScrollButtons[count]:SetScript("OnClick", FilterScrollClick)
 
 		-- set name style
@@ -274,6 +274,7 @@ function FilterScrollDoubleClick()
 	end
 end
 
+--[[
 function FilterScrollClick(self, button, down)
 
 	if down == true then return end
@@ -286,6 +287,24 @@ function FilterScrollClick(self, button, down)
 		FilterListDrawUpdate(IgnoreScrollFrame)
 	end
 end
+]]
+function FilterScrollClick(self, button, down)
+	if down then return end
+	local id = self:GetID()
+	if button == "LeftButton" then
+		if FilterScrollSelect ~= id then
+			FilterScrollSelect = id
+			FilterListDrawUpdate(FilterScrollFrame)
+		else
+			FilterScrollDoubleClick()
+		end
+	elseif button == "RightButton" then
+		if GlobalIgnoreDB.filterDesc[id] ~= nil then
+			GlobalIgnoreDB.filterActive[id] = not GlobalIgnoreDB.filterActive[id]
+			FilterListDrawUpdate(FilterScrollFrame)
+		end
+	end
+end
 
 
 --------------------------
diff --git a/Interface/AddOns/GlobalIgnoreList/GlobalIgnoreList.lua b/Interface/AddOns/GlobalIgnoreList/GlobalIgnoreList.lua
index 4a1b33c..da9dcb1 100644
--- a/Interface/AddOns/GlobalIgnoreList/GlobalIgnoreList.lua
+++ b/Interface/AddOns/GlobalIgnoreList/GlobalIgnoreList.lua
@@ -383,7 +383,7 @@ local function ResetIgnoreDB()
 		filterDesc		= {},
 		filterList		= {},
 		skipGuild		= true,
-		skipParty		= false,
+		skipParty		= true,
 		skipPrivate		= true
 	}
 	
@@ -632,47 +632,163 @@ local function ApplicationStartup(self)
 	ShowMsg(L["LOAD_1"])
 	
 	-- Set filter defaults
-	
-	filterDefDesc[#filterDefDesc + 1]     = "Filter \"Anal\" Spammers"
-	filterDefFilter[#filterDefFilter + 1] = "([word=anal] or [contains=analan]) and ([link] or [words=2])"
-	filterDefActive[#filterDefActive + 1] = true
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Thunderfury linking"
-	filterDefFilter[#filterDefFilter + 1] = "[item=19019]"
+	filterDefDesc[#filterDefDesc + 1]     = "金商"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=收] or [contains=出] or [contains=卖]) and ([contains=米] or [contains=白菜]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = true
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Gold Spam #1"
-	filterDefFilter[#filterDefFilter + 1] = "([contains=.c0m] or [contains=.c.0.m] or [contains=,com] or ([contains=.com] and ([contains=code] or [contains=usd] or [contains=+15]))"	
+	filterDefDesc[#filterDefDesc + 1]     = "奸商"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=收]) and ([contains=邮寄] or [contains=邮件] or [contains=邮箱] or [contains=到付]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = true
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Gold Spam #2"
-	filterDefFilter[#filterDefFilter + 1] = "([contains=░] or [contains=▒] or [contains=▓] or [contains=█]) and ([contains=wts] or [contains=sell] or [contains=gold] or [contains=share]))"
+	filterDefDesc[#filterDefDesc + 1]     = "布"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=符文布] or [contains=魔纹布] or [contains=丝绸]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "皮"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=硬甲皮] or [contains=厚皮]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "草药"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=梦叶草] or [contains=格罗姆之血] or [contains=墓地苔]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "附魔"
+	filterDefFilter[#filterDefFilter + 1] = "(([contains=附魔] or [contains=FM]) or (([contains=力] or [contains=敏] or [contains=火抗]) and ([contains=十字军] or [contains=灼热] or [contains=冰寒]))) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "飞机"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=飞机] or [contains=机票] or [contains=航空] or [contains=航班]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = true
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Gold Spam #3"
-	filterDefFilter[#filterDefFilter + 1] = "([contains=deliver] or [contains=delivery] or [contains=gold] or [contains=delievery] or [contains=sale]) and ([contains=.com] or [contains=,com] or [contains=c0m])"
+	filterDefDesc[#filterDefDesc + 1]     = "金团"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=金团] or [contains=G团] or [contains=内销]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "公会招募"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=收] or [contains=招] or [contains=活动] or [contains=时间]) and ([contains=公会] or [contains=工会]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = true
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Gold Spam #4"
-	filterDefFilter[#filterDefFilter + 1] = "([contains=wts] or [contains=sell]) and [contains=share] and ([contains=account] or [contains=acc]))"
+	filterDefDesc[#filterDefDesc + 1]     = "带刷"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=带刷] or [contains=有偿] or [contains=监狱] or [contains=血色] or [contains=墓地] or [contains=教堂] or [contains=STSM] or [contains=后门] or [contains=厄运]) and ([contains=带] or [contains=法] or [contains=次] or [contains=消费] or [contains=老板]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = true
-	
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Guild Recruitment"
-	filterDefFilter[#filterDefFilter + 1] = "(([contains=<] and [contains=>]) or ([contains=\\[] and [contains=\\]]) or ([contains=\\(] and [contains=\\)])) and ([contains=recruit] or [contains=progress] or [contains=raid] or [contains=guild] or [contains=seek] or [contains=mythic])"
+
+	filterDefDesc[#filterDefDesc + 1]     = "AA队"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=AA] and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = false
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Community Recruitment"
-	filterDefFilter[#filterDefFilter + 1] = "[community]"
+	filterDefDesc[#filterDefDesc + 1]     = "纳克萨玛斯"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=纳克萨玛斯] or [contains=NAXX]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "安其拉"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=安其拉] or [contains=TAQ]) and not ([contains=废墟]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "安其拉废墟"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=废墟] or [contains=RAQ]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "祖尔格拉布"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=祖尔格拉布] or [contains=祖格] or [contains=ZG]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "黑翼之巢"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=黑翼] or [contains=BWL]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "奥妮克希亚的巢穴"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=黑龙] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "熔火之心"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=熔火之心] or [contains=MC]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "黑石塔上层"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=黑上] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "黑石塔下层"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=黑下] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "厄运之槌"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=厄运] or [contains=DM] or [contains=贡品]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = false
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter WTS"
-	filterDefFilter[#filterDefFilter + 1] = "[contains=WTS] or [contains=WTB]"
+	filterDefDesc[#filterDefDesc + 1]     = "通灵学院"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=通灵] or [contains=TL] or [contains=院长]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = false
 
-	filterDefDesc[#filterDefDesc + 1]     = "Filter Chinese/Korean/Japanese"
-	filterDefFilter[#filterDefFilter + 1] = "[nonlatin]"
+	filterDefDesc[#filterDefDesc + 1]     = "斯坦索姆"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=斯坦索姆] or [contains=STSM] or [contains=DK]) and not [channel=s]"
 	filterDefActive[#filterDefActive + 1] = false
 
+	filterDefDesc[#filterDefDesc + 1]     = "黑石深渊"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=黑石] or [contains=深渊]) and not ([contains=塔] or [contains=黑暗]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "沉没的神庙"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=神庙]) and not ([contains=安其拉]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "祖尔法拉克"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=祖尔] or [contains=祖法] or [contains=ZUL] or [contains=ZR]) and not ([contains=格拉布]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "玛拉顿"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=玛拉顿] or [contains=MLD]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "奥达曼"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=奥达曼] or [contains=ADM] or [contains=矮人本]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "剃刀高地"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=剃刀高地] or [contains=TDGD]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "剃刀沼泽"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=剃刀沼泽] or [contains=TDZZ]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "血色修道院"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=血色] or [contains=XS] or [contains=墓地] or [contains=图书馆] or [contains=军械库] or [contains=武器库] or [contains=教堂]) and not ([contains=斯坦索姆] or [contains=STSM] or [contains=区]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "诺莫瑞根"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=诺莫瑞根] or [contains=侏儒本]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "监狱"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=监狱] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "黑暗深渊"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=黑暗深渊] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "影牙城堡"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=影牙] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "哀嚎洞穴"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=哀嚎] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "死亡矿井"
+	filterDefFilter[#filterDefFilter + 1] = "([contains=死矿] or [contains=SK] or [contains=矿井]) and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "怒焰裂谷"
+	filterDefFilter[#filterDefFilter + 1] = "[contains=怒焰] and not [channel=s]"
+	filterDefActive[#filterDefActive + 1] = false
+
+	filterDefDesc[#filterDefDesc + 1]     = "暴雪群组"
+	filterDefFilter[#filterDefFilter + 1] = "[community]"
+	filterDefActive[#filterDefActive + 1] = true
+
 	faction = UnitFactionGroup("player")
 		
 	if GlobalIgnoreDB == nil then	
@@ -686,7 +802,7 @@ local function ApplicationStartup(self)
 	end
 
 	if GlobalIgnoreDB.skipParty == nil then
-		GlobalIgnoreDB.skipParty = false
+		GlobalIgnoreDB.skipParty = true
 	end
 
 	if GlobalIgnoreDB.skipGuild == nil then
@@ -1299,7 +1415,7 @@ function filterComplex (filterStr, chatStr, chNum)
 								result = result .. "F"									
 							end
 						elseif token == "[channel]" then
-							if tonumber(tokenData) == chNum then
+							if tokenData == tostring(chNum) then
 								result = result .. "T"
 							else
 								result = result .. "F"
@@ -1569,6 +1685,12 @@ local function chatMessageFilter (self, event, message, from, t1, t2, t3, t4, t5
 		if message == ERR_IGNORE_FULL then
 			return true
 		end
+	elseif event == "CHAT_MSG_SAY" then chnum = "s"
+	elseif event == "CHAT_MSG_YELL" then chnum = "y"
+	elseif event == "CHAT_MSG_PARTY" then chnum = "p"
+	elseif event == "CHAT_MSG_RAID" then chnum = "r"
+	elseif event == "CHAT_MSG_GUILD" then chnum = "g"
+	elseif event == "CHAT_MSG_OFFICER" then chnum = "o"
 	end
 			
 	if GIL_Loaded ~= true then
