diff --git a/Interface/AddOns/NugEnergy/NugEnergy.lua b/Interface/AddOns/NugEnergy/NugEnergy.lua
index 7bf3613..20aa05b 100644
--- a/Interface/AddOns/NugEnergy/NugEnergy.lua
+++ b/Interface/AddOns/NugEnergy/NugEnergy.lua
@@ -65,7 +65,7 @@ local UnitAura = UnitAura
 
 local defaults = {
     point = "CENTER",
-    x = 0, y = 0,
+    x = -150, y = -75,
     marks = {},
     rage = true,
     energy = true,
