diff --git a/Interface/AddOns/Quartz/Quartz.lua b/Interface/AddOns/Quartz/Quartz.lua
index 3ea4922..ce8b556 100644
--- a/Interface/AddOns/Quartz/Quartz.lua
+++ b/Interface/AddOns/Quartz/Quartz.lua
@@ -28,7 +28,7 @@ local type, pairs, tonumber = type, pairs, tonumber
 
 local defaults = {
 	profile = {
-		modules = { ["*"] = true, ["EnemyCasts"] = false },
+		modules = { ["*"] = true, ["EnemyCasts"] = false, ["Buff"] = false, ["Target"] = false },
 		hidesamwise = true,
 		sparkcolor = {1, 1, 1, 0.5},
 		spelltextcolor = {1, 1, 1},
