diff --git a/Interface/AddOns/RealMobHealth/UITweaks.lua b/Interface/AddOns/RealMobHealth/UITweaks.lua
index 4eab7b2..da1c63c 100644
--- a/Interface/AddOns/RealMobHealth/UITweaks.lua
+++ b/Interface/AddOns/RealMobHealth/UITweaks.lua
@@ -15,7 +15,7 @@ AddOn.Options=AddOn.Options or {};
 ----------------------------------
 AddOn.Options.ShowStatusBarTextAdditions=true;
 AddOn.Options.ModifyHealthBarText=true;
-AddOn.Options.ShowTooltipText=true;
+AddOn.Options.ShowTooltipText=false;
 AddOn.Options.ShowTooltipHealthText=true;
 AddOn.Options.ShowNamePlateHealthText=true;
 
