diff --git a/Interface/AddOns/SimpleRaidTargetIcons/SimpleRaidTargetIcons.lua b/Interface/AddOns/SimpleRaidTargetIcons/SimpleRaidTargetIcons.lua
index deede61..48da7ed 100644
--- a/Interface/AddOns/SimpleRaidTargetIcons/SimpleRaidTargetIcons.lua
+++ b/Interface/AddOns/SimpleRaidTargetIcons/SimpleRaidTargetIcons.lua
@@ -167,16 +167,16 @@ srti.defaults = {
 	["ctrl"] = true,
 	["alt"] = false,
 	["shift"] = false,
-	["singlehover"] = false,
+	["singlehover"] = true,
 
-	["double"] = true,
+	["double"] = false,
 	["speed"] = 0.25,
 	["radialscale"] = 1.0,
-	["doublehover"] = false,
+	["doublehover"] = true,
 
-	["bindinghover"] = false,
+	["bindinghover"] = true,
 
-	["hovertime"] = 0.2,
+	["hovertime"] = 0.1,
 }
 
 -- srti.frame
