diff --git a/Interface/AddOns/TomTom/TomTom.lua b/Interface/AddOns/TomTom/TomTom.lua
index a1a659c..18bd9cc 100644
--- a/Interface/AddOns/TomTom/TomTom.lua
+++ b/Interface/AddOns/TomTom/TomTom.lua
@@ -34,7 +34,7 @@ function TomTom:Initialize(event, addon)
                 corpse_arrow = true,
             },
             block = {
-                enable = true,
+                enable = false,
                 accuracy = 2,
                 bordercolor = {1, 0.8, 0, 0.8},
                 bgcolor = {0, 0, 0, 0.4},
@@ -45,10 +45,10 @@ function TomTom:Initialize(event, addon)
                 throttle = 0.2,
             },
             mapcoords = {
-                playerenable = true,
+                playerenable = false,
                 playeraccuracy = 2,
                 playeroffset = 0,
-                cursorenable = true,
+                cursorenable = false,
                 cursoraccuracy = 2,
                 cursoroffset = 0,
 				throttle = 0.1,
diff --git a/Interface/AddOns/TomTom/TomTom_Config.lua b/Interface/AddOns/TomTom/TomTom_Config.lua
index aed9674..6149b8d 100644
--- a/Interface/AddOns/TomTom/TomTom_Config.lua
+++ b/Interface/AddOns/TomTom/TomTom_Config.lua
@@ -346,7 +346,7 @@ local function createconfig()
 						desc = L["Resets the position of the waypoint arrow if its been dragged off screen"],
 						func = function()
 							TomTomCrazyArrow:ClearAllPoints()
-							local pos = {"CENTER", nil , "CENTER", 0, 0}
+							local pos = {"TOP", nil , "TOP", 0, -50}
 							set({arg="arrow.position"}, pos)
 							TomTomCrazyArrow:SetPoint(pos[1], UIParent, pos[3], pos[4], pos[5])
 						end,
diff --git a/Interface/AddOns/TomTom/TomTom_CrazyArrow.lua b/Interface/AddOns/TomTom/TomTom_CrazyArrow.lua
index 769435c..2dfcd03 100644
--- a/Interface/AddOns/TomTom/TomTom_CrazyArrow.lua
+++ b/Interface/AddOns/TomTom/TomTom_CrazyArrow.lua
@@ -76,7 +76,7 @@ local function OnEvent(self, event, ...)
 	if (event == "PLAYER_ENTERING_WORLD") then
         wayframe:ClearAllPoints()
         if not TomTom.profile.arrow.position then
-            TomTom.profile.arrow.position = {"CENTER", nil , "CENTER", 0, 0}
+            TomTom.profile.arrow.position = {"TOP", nil , "TOP", 0, -50}
         end
         local pos = TomTom.profile.arrow.position
         wayframe:SetPoint(pos[1], UIParent, pos[3], pos[4], pos[5])
