diff --git a/Interface/AddOns/TrackOMatic/TrackOMatic.lua b/Interface/AddOns/TrackOMatic/TrackOMatic.lua
index a196afe..1e19d6b 100644
--- a/Interface/AddOns/TrackOMatic/TrackOMatic.lua
+++ b/Interface/AddOns/TrackOMatic/TrackOMatic.lua
@@ -409,7 +409,7 @@ function TrackOMatic_LoadSettings()
         --TrackOMatic_Upgrade_11000(version);
     end
     TRACKOMATIC_VARS['config']['version'] = TrackOMatic.VersionID;
-    TrackOMatic_CheckSetting("texture", "LiteStep");
+    TrackOMatic_CheckSetting("texture", "Minimalist");
     TrackOMatic_CheckSetting("gold_glow_negative", true);
     TrackOMatic_CheckSetting("show_lowest_durability", false);
     TrackOMatic_CheckSetting("item_default_glow", true);
@@ -426,7 +426,7 @@ function TrackOMatic_LoadSettings()
     TrackOMatic_CheckSetting("show_alert_boxes", true);
     local upd = TrackOMatic_CheckSetting("update_interval", 1);
 
-    TrackOMatic_CheckProfileSetting("width", 100);
+    TrackOMatic_CheckProfileSetting("width", 75);
     TrackOMatic_CheckProfileSetting("enable_glows", true);
     TrackOMatic_CheckProfileSetting("locked", false);
     TrackOMatic_CheckProfileSetting("visible", true);
diff --git a/Interface/AddOns/TrackOMatic/config/Config.lua b/Interface/AddOns/TrackOMatic/config/Config.lua
index fb4cd78..7cc922d 100644
--- a/Interface/AddOns/TrackOMatic/config/Config.lua
+++ b/Interface/AddOns/TrackOMatic/config/Config.lua
@@ -268,8 +268,8 @@ function TrackOMatic_Config_SetDefaults()
     TRACKOMATIC_VARS['config']['show_load_message'] = false;
     TRACKOMATIC_VARS['config']['show_alert_boxes'] = true;
     TrackOMatic_SetScale(1);
-    TrackOMatic_SetWidth(100);
-    TrackOMatic_Config_SetTexture("LiteStep");
+    TrackOMatic_SetWidth(75);
+    TrackOMatic_Config_SetTexture("Minimalist");
     TRACKOMATIC_VARS['config']['gold_glow_negative'] = true;
     TRACKOMATIC_VARS['config']['show_lowest_durability'] = false;
     TRACKOMATIC_VARS['config']['durability_glow_broken'] = true;
